modules/src/mock/context.rs

```diff
@@ -225,8 +224,9 @@ impl MockContext {
     }
 
     /// A datagram passes from the relayer to the IBC module (on host chain).
+    /// Alternative method to `ICS18Context::send` that does not exercise any serialization.
     /// Used in testing the ICS18 algorithms, hence this may return a ICS18Error.
-    fn recv(&mut self, msg: ICS26Envelope) -> Result<(), ICS18Error> {
+    pub fn recv(&mut self, msg: ICS26Envelope) -> Result<(), ICS18Error> {
```

Comment:
maybe change the name?

Reply:
Good idea
https://github.com/informalsystems/ibc-rs/pull/422/commits/c92fb0ad22eb3b52e70f66210189c50461732565

Reply:
Good idea
https://github.com/informalsystems/ibc-rs/pull/422/commits/c92fb0ad22eb3b52e70f66210189c50461732565